.coveragerc

```diff
@@ -7,7 +7,7 @@ source =
     Orange
 omit =
     Orange/canvas/*
-    Orange/tests/*
+     */test*
```

Comment:
I agree, but could we use `*/tests/*` instead of `*/test*`? `*/test*` covers `Orange/evaluation/testing.py`, which is an ordinary module.

If `*/tests/*` doesn't suffice, we can also exclude `*/test_*`. I suppose no module will start with `test_`.

Reply:
Good catch. Thanks!